internal/uvm/create_lcow.go

```diff
@@ -402,6 +399,7 @@ func CreateLCOW(ctx context.Context, opts *OptionsLCOW) (_ *UtilityVM, err error
 	}
 
 	if opts.UseGuestConnection && opts.ExternalGuestConnection {
+		log.G(ctx).Debugf("Using external GCS bridge for UVM: %s", uvm.runtimeID)
```

Comment:
It will be easier to parse this data if you add the runtime ID as a separate field.

e.g.
```
log.G(ctx).WithField("vmID", uvm.runtimeID).Debug("Using external GCS bridge for UVM")
```